lib/broccoli/ember-app-with-packages.js

```diff
@@ -71,7 +71,7 @@ function EmberAppWithPackages(defaults, options) {
         templates: new Funnel(appBuildConfig.appTreesBasePath + '/templates')
       }
     };
-    this.bootApp = new EmberApp(configWithTrees, defaults, sharedBuildConfig, appBuildConfig);
+    this.bootApp = new EmberApp(configWithTrees, sharedBuildConfig, appBuildConfig, defaults);
```

Comment:
I had another look and it turns out that `ember-cli-htmlbars-inline-precompile` has a [check to only register the babel plugin once](https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/blob/master/index.js#L42), in our case we need it once for each package, but only the first app gets it. To work around this we have to get the options manually. 

In simple cases, pushing defaults to the last param will do this, but it can also have some other side effects, like default options overriding custom options, so I want to avoid that. 

I had to create a [separate PR](https://github.com/MiguelMadero/ember-cli-bundle-loader/pull/9), can you check that it address your issue?